ui.py

```diff
@@ -56,12 +60,15 @@ def hide(self) -> None:
 
     def show(self, region: Tuple[int, int], completions: List[CopilotPayloadCompletion], cycle: int = 0) -> None:
         cycle %= len(completions)
-        display_text = self.get_display_text(region, completions[cycle]["displayText"])
 
+        completion_uuid = completions[cycle]["uuid"]
+
+        display_text = self.get_display_text(region, completions[cycle]["displayText"])
```

Comment:
Sorry for nitpicking, but maybe we could just introduce a variable `completion` and use it here and below
```suggestion
        completion = completions[cycle]

        display_text = self.get_display_text(region, completion["displayText"])
```